test/unit/oidc-profile.ts

```diff
@@ -4,66 +4,66 @@ import { gateway as MoltinGateway } from '../../src/moltin'
 
 const apiUrl = 'https://api.moltin.com/v2'
 
-describe('Authentication Profiles', () => {
+describe('Oidc Profiles', () => {
   const Moltin = MoltinGateway({
     client_id: 'XXX'
   })
 
   const realmId = '96764ca9-af12-4355-acce-37fa2ef4728a'
 
-  it('Get all Profiles', () => {
+  it('Get all Oidc Profiles', () => {
     nock(apiUrl, {})
-      .get(/\/authentication-realms\/(.*)\/profiles\/oidc/)
+      .get(/\/authentication-realms\/(.*)\/oidc-profiles/)
       .reply(200, {})
 
-    Moltin.AuthenticationProfile.All(
+    Moltin.OidcProfile.All(
       realmId
     ).then(res => {
       assert.isObject(res)
     }).catch(err=>console.error(err))
   })
 
-  it('Get a single Profile', () => {
+  it('Get a single Oidc Profile', () => {
     nock(apiUrl, {})
-      .get(/\/authentication-realms\/(.*)\/profiles\/oidc\/(.*)/)
+      .get(/\/authentication-realms\/(.*)\/oidc-profiles\/(.*)/)
       .reply(200, {})
 
-    Moltin.AuthenticationProfile.Get({
+    Moltin.OidcProfile.Get({
       realmId,
       profileId: '4da65e78-7f9b-4248-b498-823d43120da9'
     }).then(res => {
       assert.isObject(res)
     }).catch(err=>console.error(err))
   })
 
-  it('Create a single authentication Profile', () => {
+  it('Create a single Oidc Profile', () => {
     nock(apiUrl, {})
-      .post(/\/authentication-realms\/(.*)\/profiles\/oidc/)
+      .post(/\/authentication-realms\/(.*)\/oidc-profiles/)
       .reply(201, {})
 
     const body = {
       name: 'Keycloak',
-      type: 'authentication-profile-oidc',
+      type: 'oidc-profile-oidc',
```

Comment:
is it really meant to be 'oidc-profile-oidc'? just checking

Reply:
Good catch @razkevich. This seems to have been find/replaced globally